mscore/data/shortcuts.xml

```diff
@@ -491,6 +491,14 @@
     <seq>+</seq>
     </SC>
   <SC>
+    <key>pad-raise</key>
+    <seq>NumPad++</seq>
```

Comment:
It didn't tell me there was a collision when I set it manually.
The arrow-up/arrow-down key work differently (follow a chromatic scale instead of adding an accidental, and they don't toggle).